Yoast/a11y-speak
# Add simple replace to strip HTML tags.

Fixes #9 

Adds a simple replace to strip _most_ of the possible HTML in the message strings. I guess messages dispatched to A11ySpeak won't contain complex HTML so this should be enough.

How to test:
for example on https://github.com/Yoast/wordpress-seo/issues/5909 insert a wrong variable in a template field and ensure the string sent to A11ySpeak doesn't contain HTML. For example this:

`Warning: the variable <code>userid</code> cannot be used in this template. See the help tab for more info.`

When inserted in the aria live region should become this:

`Warning: the variable userid cannot be used in this template. See the help tab for more info.`
